tests/test_main.py

```diff
@@ -193,7 +193,7 @@ def test_create_license_in_use_dont_exists(client):
         json={"license_name": "test_name", "quantity": 1, "user_name": "user1", "lead_host": "host1"},
```

Comment:
```suggestion
        json={"license_name": "test_name", "quantity": 1, "user_name": "user1", "lead_host": "host1"},
```

Does this need to match the format `<product>.<feature>`?

Reply:
No, just the feature